arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1040,6 +1040,10 @@
             "$ref": "#/definitions/IpFilterRule"
           }
         },
+        "provisioningState": {
+          "description": "The provisioning state.",
+          "type": "string"
```

Comment:
With the assumption that this is a server generated value, this should be marked as read only.